Control coverage/Feature/CloudService.md

```diff
@@ -4,3 +4,5 @@
 | <b>ControlId:</b><br>Azure_CloudService_SI_Disable_RemoteDesktop_Access<br><b>DisplayName:</b><br>Disable Remote Desktop (RDP) access on cloud service roles<br><b>Description: </b><br>Disable Remote Desktop (RDP) access on cloud service roles. | <b>ARM API to get the list of Deployment Slots in a Cloud Service:</b><br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/<br>Microsoft.ClassicCompute/domainNames/{cloudServiceName}/<br>deploymentSlots?api-version=2016-11-01<br><b>Properties:</b><br>properties.slotType, properties.configuration<br><b>ARM API to get the list of Cloud Service Roles in a Deployment Slot:</b><br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/<br>Microsoft.ClassicCompute/domainNames/{cloudServiceName}/<br>slots/{slotName}/roles?api-version=2016-04-01<br><b>Properties:</b><br>[\*].name<br><b>ARM API to get the list of Extensions in a Cloud Service Role:</b><br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/<br>Microsoft.ClassicCompute/domainNames/{cloudServiceName}/<br>slots/{slotName}/roles/{roleName}/extensionReferences?api-version=2015-06-01<br><b>Properties:</b><br>[\*].properties.name, [\*].properties.state | <b>Passed:</b><br>Remote Desktop access is not enabled for any Cloud Service Role.<br><b>Failed:</b><br>Remote Desktop access is enabled for Cloud Service Role(s). |
+| <b>ControlId:</b><br>Azure_CloudService_DP_DontAllow_HTTP_Access_InputEndpoints<br><b>DisplayName:</b><br>Encrypt data in transit for Cloud service role <br><b>Description: </b><br> Cloud Service must only be accessible over HTTPS. Enable https for InputEndpoints.| <b> ARM API to get Cloud Service roles details: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ClassicCompute/domainNames/{cloudServiceName}/slots/{slotName}/roles/{roleName}?api-version=2016-04-01  <br><b>Properties:</b><br> name: 'properties.inputEndpoints.protocol| <b>Passed: </b><br>No active non SSL enabled input endpoints for all the roles in cloud service. <br><b>Failed: </b><br>Active non SSL enabled input endpoints are present for any of the roles in cloud service. |
```

Comment:
Looks like there is typo in Properties [name: 'properties.inputEndpoints.protocol]